R/calcIEnvPolicies.R

```diff
@@ -0,0 +1,77 @@
+#' calcIEnvPolicies
+#'
+#' Use carbon price data from the EU Reference Scenario 2020 and from the
```

Comment:
readENGAGE uses data from the ENGAGE project, that is the origin of the GP_CurPol_T45 scenario. maybe worth mentioning ENGAGE

Reply:
ok